build.sbt

```diff
@@ -135,20 +135,22 @@ lazy val dumpEventSchema = (project in file("sbt-dump-event-schema"))
       Seq(
         "-Xmx1024M",
         "-Dplugin.version=" + version.value,
-        "-Dcompiler-plugin.version=" + (dumpEventSchemaCompilerPlugin.componentProjects.head / version).value)
+        "-Dcompiler-plugin.version=" + (dumpPersistenceSchemaCompilerPlugin.componentProjects.head / version).value)
     },
     scriptedDependencies := { // publishing compiler plugin locally for testing
       scriptedDependencies.value
-      (dumpEventSchemaCompilerPlugin
-        .filterProjects(Seq(ScalaVersionAxis(scalaVersion212, "2.12")))
-        .head / publishLocal).value
-      (dumpEventSchemaCompilerPlugin
-        .filterProjects(Seq(ScalaVersionAxis(scalaVersion213, "2.13")))
-        .head / publishLocal).value
+      (dumpPersistenceSchemaCompilerPlugin.projectRefs.head / publishLocal).value
```

Comment:
Wow can you drop a comment why this uncanny split b/w `.head`/`.tail.head`? it's still about 2.12/2.13 right?